### PR TITLE
Remove wrong annotation

### PR DESCRIPTION
**Pull request by TobiGr**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
The current release PR is failing the sonar lint quality quard. This PR removes the wrong annotation to make sonar lint happy:
> Do not use `@Nullable` in Kotlin; the nullability is determined by the Kotlin type `View.OnClickListener` **not** ending with `?` which declares it not nullable, contradicting the annotation

 (see https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=9869&id=TeamNewPipe_NewPipe&open=AYacpIb_J93zs3Jn_4eI)

#### APK testing

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by AudricV** [comment]
> ```
> > Task :app:runKtlint
> /home/runner/work/NewPipe/NewPipe/app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt:9:1: Unused import (no-unused-imports)
> 
> > Task :app:runKtlint FAILED
> ```

You need to remove the annotation import too.

File: app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt [hunk 1]
```diff
@@ -144,7 +144,7 @@ class ErrorPanelHelper(
      */
     private fun showAndSetErrorButtonAction(
         @StringRes resid: Int,
-        @Nullable listener: View.OnClickListener
+        listener: View.OnClickListener?
```
[the remark this answers is not in the document]
Comment (Stypox) [reply app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt:147]: Test

**Review by Stypox — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (TobiGr) [reply app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt:146]: According to https://developer.android.com/reference/android/view/View#setOnClickListener(android.view.View.OnClickListener the listener can be null

[the remark this answers is not in the document]
Comment (Stypox) [reply app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt:146]: Yes, that's the point. That function shows a button, and a button without listener does not make sense.

**Pull request merged**